packaging/language/composer.py

```diff
@@ -116,6 +132,8 @@ def parse_out(string):
     return re.sub("\s+", " ", string).strip()
 
 def has_changed(string):
+    if string == "":
```

Comment:
Sometimes composer returns nothing, but ansible thinks that something changed, because empty string doesn't have string 'Nothing to install...'.

Maybe this condition should be improved somehow, but i'm not sure.